src/mlang/backend_ir/bir_roundops.ml

```diff
@@ -14,6 +14,8 @@
    You should have received a copy of the GNU General Public License along with
    this program. If not, see <https://www.gnu.org/licenses/>. *)
 
+let epsilon = 0.000001
```

Comment:
So this value should be cutsomizable, and we can piggyback on the existing `--test_error_margin` CLI parameter here : https://github.com/MLanguage/mlang/blob/master/src/mlang/utils/cli.ml#L174